internal/service/advancedcluster/resource_advanced_cluster_test.go

```diff
@@ -345,6 +352,13 @@ func TestAccClusterAdvancedCluster_advancedConfig(t *testing.T) {
 		processArgsUpdated = &admin.ClusterDescriptionProcessArgs20240805{
 			DefaultMaxTimeMS: conversion.IntPtr(65),
 			ChangeStreamOptionsPreAndPostImagesExpireAfterSeconds: conversion.IntPtr(100),
+			TlsCipherConfigMode:            conversion.StringPtr("CUSTOM"),
+			CustomOpensslCipherConfigTls12: &[]string{"TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256", "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384"},
```

Comment:
[nit] refactor to const and extract. Then we can do checks of the actual values not only the count